docs/content/importing/_index.md

```diff
@@ -0,0 +1,70 @@
++++
+title="Importing Code"
+weight=2
++++
+
+Code can be imported into joern in two ways. The program `joern-parse`
+can be used to create code property graphs and persist them in a
+format based on Google Protocol Buffer messages, or, `create-cpg` can
+be used to instruct joern-server to create a code property graph and
```

Comment:
not sure about the rest of these docs, but don't we normally write it as one word, i.e. `codepropertygraph`?

Reply:
`codepropertygraph` is the repo. Maybe it should be Code Property Graphs though (capital letters) since it's a name. I think I'll do that.